spalp.libraries.yml

```diff
@@ -0,0 +1,18 @@
+json-form:
+  remote: https://github.com/brutusin/json-forms
+  version: VERSION
+  license:
```

Comment:
looking at the example on https://www.drupal.org/project/drupal/issues/2276219, we need to link to the remote license as well